x-pack/docs/en/watcher/managing-watches.asciidoc

```diff
@@ -28,7 +28,7 @@ For example, the following returns the first 100 watches:
 
 [source,console]
 --------------------------------------------------
-GET .watches/_search
+GET .watches/_search?allow_system_index_access=true
```

Comment:
we should open an issue for the watcher docs since we need to stop accessing `.watches` directly

Reply:
I've added this to the list of tests that need to be adjusted to remove this flag (https://github.com/elastic/elasticsearch/issues/62501).